automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoSuite.scala

```diff
@@ -243,6 +249,46 @@ trait NewBillingProjectAndWorkspaceBeforeAndAfterAll extends BillingProjectUtils
     } else IO.unit
 }
 
+final case class AzureBillingProjectName(value: String) extends AnyVal
```

Comment:
So I have a separate test folder for azure. I think this fits well here as a suite-level mix-in (i.e multiple suites will want an azure billing provided). The code in `LeonardoTestUtils` is well placed too I believe, since splitting it up into other files will bloat the amount of things a test has to extend and make tracing down what code comes from where very hard